include/EncoderDecoder/Huffman/PaddedString.h

```diff
@@ -0,0 +1,12 @@
+#ifndef PADDED_STRING_H
+#define PADDED_STRING_H
+
+#include <string>
+
+class PaddedString {
```

Comment:
`struct` here as a convention for simple data structures

Reply:
TODO